### PR TITLE
adds new data visualizations to Cases page, makes fixes for style

### PR DESCRIPTION
**Pull request by paulstgermain**
## Description

This pull changes the data visualization on the cases page from a pie chart (ew) to a pair of bar charts for now. Additionally makes changes to the CSS for this page to display them appropriately.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/CaseTable/CaseTable.css [hunk 1]
```diff
@@ -212,3 +212,17 @@ button.ant-btn.columnsBtn:hover {
...
+  display: flex;
+}
+
+.divider {
+  width: 2px;
+  height: 200px;
+  background-color: grey;
+  border: 1px solid black;
+  margin-top: 2rem;
+}
```
Comment (katieolson84) [inline src/components/pages/CaseTable/CaseTable.css:228]: I would like to convert this to a less file in the future to keep the styling consistent and then you can nest the styling as well.

File: src/components/pages/CaseTable/CaseTable.js [hunk 2]
```diff
@@ -494,29 +497,79 @@ export default function CaseTable(props) {
     });
 
     return (
+      // <Plot
```
Comment (katieolson84) [inline src/components/pages/CaseTable/CaseTable.js:500]: Did you want to leave this code in for reference later? Or is it needed?

File: src/components/pages/CaseTable/CaseTable.js [hunk 3]
```diff
@@ -494,29 +497,79 @@ export default function CaseTable(props) {
...
-              'Granted',
-              'Denied',
-              'Remanded',
-              'Sustained',
-              'Terminated',
-            ],
-            textinfo: 'label+percent',
-            textposition: 'outside',
-            automargin: true,
+            type: 'bar',
```
Comment (katieolson84) [inline src/components/pages/CaseTable/CaseTable.js:527]: Bye Bye pie charts. Yay!

File: src/components/pages/CaseTable/CaseTable.js (hunk 2 above)
Comment (paulstgermain) [reply src/components/pages/CaseTable/CaseTable.js:500]: Yup, this code is a reference for later, will be deleting it soon!

File: src/components/pages/CaseTable/CaseTable.js [hunk 4]
```diff
@@ -494,29 +497,79 @@ export default function CaseTable(props) {
...
+            y: [granted, denied, remanded, sustained, terminated],
           },
         ]}
-        layout={{
-          height: 300,
-          width: 300,
-          showlegend: false,
-          title: 'Decision Rate',
-        }}
+        layout={{ width: 500, height: 300, title: 'Decision Rate' }}
```
Comment (katieolson84) [inline src/components/pages/CaseTable/CaseTable.js:532]: This code looks so clean, nice job!

File: src/components/pages/CaseTable/CaseTable.css (hunk 1 above)
Comment (paulstgermain) [reply src/components/pages/CaseTable/CaseTable.css:228]: You got it - I'll switch it over now and make another push!

**Review by katieolson84 — CHANGES_REQUESTED** [review]
The code looks nice and clean. I did not approve yet to see if you needed the commented code left in. Great work!

File: src/components/pages/CaseTable/CaseTable.js (hunk 4 above)
Comment (MichelleCCodes) [reply src/components/pages/CaseTable/CaseTable.js:532]: Thanks for working with DS to get this terminology down!

**Review by MichelleCCodes — APPROVED** [review]
Logic looks correct, and great move to switch over to bar graph for more accurate representation!

File: src/components/pages/CaseTable/CaseTable.js (hunk 2 above)
Comment (katieolson84) [reply src/components/pages/CaseTable/CaseTable.js:500]: Perfect! Thanks for clarifying!

**Review by katieolson84 — APPROVED** [review]
Looks great!

**Pull request merged**